index.js

```diff
@@ -0,0 +1,71 @@
+// @ts-check
+
+import { getInput, setFailed } from '@actions/core'
+import { getExecOutput } from '@actions/exec'
+import { Octokit } from '@octokit/action'
+
+import { readFileSync } from 'node:fs'
+import { env } from 'node:process'
+import parseGitDiff from 'parse-git-diff'
+
+const octokit = new Octokit({
+  userAgent: 'suggest-changes',
+})
+
+const [owner, repo] = String(env.GITHUB_REPOSITORY).split('/')
+const eventPayload = JSON.parse(
```

Comment:
if you like, you can add [`@octokit/webhooks-types`](https://github.com/octokit/webhooks#importing-types) as a dev dependency and then set the type for eventPayload

```suggestion
/** @type {import("@octokit/webhooks-types").PullRequestEvent} */
const eventPayload = JSON.parse(
```

Reply:
Excellent suggestion. Thank you! ✨